src/schemas/channel.schema.ts

```diff
@@ -2,4 +2,24 @@ import * as Joi from 'joi';
 
 export const channelSchema = Joi.object({
   channelName: Joi.string().required(),
-});
\ No newline at end of file
+});
+
+export const messageSchema = Joi.object({
```

Comment:
i'd personally spit these into their own files

Reply:
It's very makes sense. I have separated them.